File: src/structorcli/templates/discord-bot-python/bot.py
import discord
from discord.ext import commands

# Replace '!' with your desired bot prefix
bot = commands.Bot(command_prefix="!", intents=discord.Intents.default())

@bot.event
async def on_ready():
    print(f"✅ Logged in as {bot.user}")

@bot.command()
async def ping(ctx):
    await ctx.send("🏓 Pong!")

# Replace 'YOUR_TOKEN_HERE' with your bot's token
bot.run("YOUR_TOKEN_HERE")

File: src/structorcli/templates/discord-bot/bot.js
const { Client, GatewayIntentBits } = require('discord.js');
const client = new Client({ intents: [GatewayIntentBits.Guilds] });

client.once('ready', () => {
    console.log('Ready! Logged in as ' + client.user.tag);
});

client.login('YOUR_TOKEN_HERE');

// Make sure to replace YOUR_TOKEN_HERE with your Discord bot token.